Vizzuality/trase
# Feature/ecuador layers

## Pivotal Tracker
https://www.pivotaltracker.com/story/show/175092310

## Description

This PR adds Ecuador layers and fixes a bug when removing unit layers

![image](https://user-images.githubusercontent.com/9701591/95221147-6f445d80-07f7-11eb-85b7-dc68f8881a7f.png)


## Testing instructions

Select Shrimp - Ecuador and check the three geo layers: Province, Parish, and Canton. Hovers and selection should work.
The choropleth is still remaining
